lib/src/http_session.dart

```diff
@@ -1,24 +1,29 @@
+import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:http/http.dart' as http;
 import 'package:http/io_client.dart';
+import 'package:cookie_store/cookie_store.dart';
 
 class HttpSession implements IOClient {
   /// Shared http session instance
   static final shared = HttpSession();
 
-  /// Get current headers value
-  Map<String, String> get headers => _headers;
+  int maxRedirects = 15;
```

Comment:
I think we should add this variable to the `HttpSession` constructor so we can set it directly.

Reply:
Oh I meant to do that my bad

Reply:
Done